cql3/restrictions/statement_restrictions.cc

```diff
@@ -1569,5 +1594,16 @@ std::vector<query::clustering_range> statement_restrictions::get_global_index_to
     return get_single_column_clustering_bounds(options, idx_tbl_schema, *_idx_tbl_ck_prefix);
 }
 
+std::vector<query::clustering_range> statement_restrictions::get_local_index_clustering_ranges(
+        const query_options& options,
+        const schema& idx_tbl_schema) const {
+    if (!_idx_tbl_ck_prefix.has_value()) {
+        on_internal_error(
+            rlogger, "statement_restrictions::get_local_index_clustering_ranges called with unprepared index");
+    }
+
+    return get_single_column_clustering_bounds(options, idx_tbl_schema, *_idx_tbl_ck_prefix);
```

Comment:
Add a comment saying we never expect multi-column restrictions here; reference #7699.